src/plugins/intel_gna/tests/functional/shared_tests_instances/single_layer_tests/lstm_cell_basic.cpp

```diff
@@ -23,14 +23,28 @@ float clip = 0.f;
 std::vector<InferenceEngine::Precision> netPrecisions = {InferenceEngine::Precision::FP32,
                                                          InferenceEngine::Precision::FP16};
 
+// LSTMCell is decomposed to Convolution.
+// Before GNA_TARGET_3_5 weights used I16 precision only even if GNA_PRECISION was set to I8.
+// Since GNA_TARGET_3_5 weights with precision I8 are allowed for convolutions,
+// but proper results are expected for models after POT.
```

Comment:
This is to inform that for targets 3_5 and higher we cannot run tests where convolution with I8 weights are used. And that it will be possible if POT will be enabled for functional tests as it is pointed in mentioned ticket 70675.